### ISSUE TITLE
For some reason the game doesn't end

### ISSUE THREAD

**Issue by andrei-uni** [kaajjo/Libre-Sudoku#30]
![Screenshot_20230224-210112](https://user-images.githubusercontent.com/92112053/221227140-04bf5d7e-d7ac-4dfc-9d46-7e6ec1ba8f50.png)

**Comment by andrei-uni** [comment]
![Screenshot_20230224-211102](https://user-images.githubusercontent.com/92112053/221228867-f266d909-de5d-46e8-bf09-f6ba5beb1144.png)
I think I've figured it out. I've set Mistakes Checking to 'With Final Solution' on this screenshot. As u can see it doesn't match. But my placement follows the rules.

**Comment by andrei-uni** [comment]
Can it be that there's more than one solution?

**Comment by kaajjo** [comment]
> Can it be that there's more than one solution?
No, I checked with others solvers, there is only one solution, and you solved it correctly. 
So it's a problem with validating or something related to it, I'll try to reproduce it, thanks for the report
